addons-tutorial.md

```diff
@@ -0,0 +1,91 @@
+Addons are used to customize the images used for the containers. An addon has *hooks* that are
+invoked at various points in the lifecycle of a container. A good example is an addon which
+enables the SSH service inside a container to provide access from the outside for any kind
+of automation of the container.  
+
+Addons are created independently, and can be attached to individual applications.
+
+<h2 id='heading--build-your-own-addon'>Build your first addon</h2>
```

Comment:
This heading isn't needed imo. That is what the whole tutorial is about. :wink: 

I suggest to use numbered headings here, one for every step that the user should do. So:
1. Create a manifest file
2. Add a pre-start hook
3. and 4. as is